.github/workflows/people-handler.yml

```diff
@@ -0,0 +1,83 @@
+name: People Handler
+on:
+  push:
+    branches: [ main ]
+    tags: [ "people-handler-[0-9]+.[0-9]+.[0-9]+" ]
+    paths: [ "preprocessors/people-handler/**" ]
+  pull_request:
+    branches: [ main ]
+    paths: [ "preprocessors/people-handler/**" ]
+  workflow_run:
+    workflows: [ "Schemas (Trigger)" ]
+    types:
+      - completed
+  workflow_dispatch:
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: shared-reality-lab/image-preprocessor-people-handler
+jobs:
+  lint:
+    name: PEP 8 style check.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - name: Install flake8
+        run: pip install flake8
+      - name: Check with flake8
+        run: python -m flake8 ./handlers/people-handler --show-source
+  build-and-push-image:
+    name: Build and Push to Registry
+    needs: lint
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Log into GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get Correct Tags
+        run: |
+          if [[ ${{ github.ref }} =~ ^refs/tags/handler-people-handler-[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Note that this does not match tag above. I also don't think I need to explain how `handler-people-handler` could use some editing.